plugins/search/package.json

```diff
@@ -49,7 +52,9 @@
     "@backstage/test-utils": "^0.1.13",
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^11.2.5",
+    "@testing-library/react-hooks": "^7.0.0",
```

Comment:
maybe downgrade this to the version that's in use elsewhere in the repo unless you need specific newer features

(yeah i agree it's due for a concerted update)